app/src/main/java/fraktal/io/android/demo/timer/ui/TimerViewStateUI.kt

```diff
@@ -2,14 +2,21 @@ package fraktal.io.android.demo.timer.ui
 
 import androidx.compose.runtime.Stable
 import fraktal.io.android.demo.timer.domain.TimerCommand
-import fraktal.io.android.demo.timer.domain.TimerViewState
+import fraktal.io.android.demo.timer.domain.TimerEvent
+import fraktal.io.android.demo.timer.domain.TimerQueryState
+import fraktal.io.android.demo.timer.domain.TimerState
+import fraktal.io.ext.FViewModel
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.toPersistentList
 import java.util.concurrent.TimeUnit
 
+typealias TimerViewModel = FViewModel<TimerCommand,TimerState, TimerViewStateUI, TimerEvent>
```

Comment:
I am not 100% sure if the `TimerViewModel` is a good name. This model represents Command model + View model = Domain model. Maybe, `TimerModel` is a better name.

Reply:
The same applies to the `FViewModel `. Maybe, `FModel` is better

Reply:
I understand your argument, but let me explain. 
In the android world, a lot of applications use the [mvvm ](https://developer.android.com/topic/libraries/architecture/viewmodel) pattern, or its derivatives (it forced by google). 
Of course, we can do without them. But I deliberately made an android component to make it easier for mobile developers to understand the demo.

Reply:
![image](https://github.com/jQwout/fmodel-android-demo/assets/37448471/d0e03117-3280-4d00-9f0e-0ae721c42679)
at the same time, *model* can mean anything. In our case - *model*)))

At present, it appears to me that this would be the most straightforward method of integrating Fmodels with the Android community.